packages/config/src/routes.ts

```diff
@@ -14,8 +14,9 @@ export enum AppRoute {
 
   Inventory = 'inventory',
   Stock = 'stock',
-  Stocktake = 'stocktake',
+  Stocktakes = 'stocktakes',
```

Comment:
a little pedantic.. there was `Locations` `Items` `Requisitions` etc, but the `Stocktake` was singular.